moped-editor/src/components/GridTable/Filters.js

```diff
@@ -305,6 +285,13 @@ const Filters = ({
       setIsOr(isOrToggleValue);
       setFilters(filterParameters);
     } else {
+      /* Clear search params since we have no advanced filters */
+      setSearchParams((prevSearchParams) => {
+        prevSearchParams.delete(advancedSearchFilterParamName);
+        prevSearchParams.delete(advancedSearchIsOrParamName);
+
+        return prevSearchParams;
+      });
```

Comment:
here, after the user clicks apply, is where we should clear the URL search params if the filters were deleted